src/plugin.ts

```diff
@@ -212,11 +212,47 @@ export function openGraphImagePlugin(options: Options): Plugin {
           const optimizedImageBuffer = await optimizeImageBuffer.toBuffer()
           const imageName = `${data.name}.${format}`
 
+          const outputPath = await fromOutputDirectory(imageName)
+
           images.push({
             name: imageName,
-            path: await fromOutputDirectory(imageName),
+            path: outputPath,
             content: optimizedImageBuffer,
           })
+
+          // check if outputPath is somewhere in `public` directory
+          if (outputPath.includes("public")) {
```

Comment:
checking if the `outputDirectory` actually belong to the `public` directory, as users could very well set it directly to the `build` directory in the plugin options, and this whole thing perhaps will not be required